lib/tasks/data.rake

```diff
@@ -6,4 +6,20 @@ namespace :data do
       UpdateSchoolsDataFromSourceJob.perform_later
     end
   end
+
+  desc 'Backfill AuditData for past vacancy.publish events'
+  namespace :backfill do
+    namespace :audit_data do
+      task vacancy_publishing: :environment do
+        PublicActivity::Activity.where(key: 'vacancy.publish').map do |audit|
+          vacancy = Vacancy.find(audit.trackable_id)
+
+          row = VacancyPresenter.new(vacancy).to_row
```

Comment:
It probably doesn't matter too much, but do we need to initialize this variable here? It only gets used if the `AuditData` doesn't exist

Reply:
We sure don't need it until after the `if`, moving it 👍